README.md

```diff
@@ -31,7 +34,8 @@ Triage is based on stack trace comparison from [gdb-command](https://github.com/
 `casr-afl` is used to triage crashes found by [AFL++](https://github.com/AFLplusplus/AFLplusplus).
 `casr-libfuzzer` can triage crashes found by
 [libFuzzer](https://www.llvm.org/docs/LibFuzzer.html) based fuzzer
-(C/C++/[go-fuzz](https://github.com/dvyukov/go-fuzz)/[Atheris](https://github.com/google/atheris)).
+(C/C++/[go-fuzz](https://github.com/dvyukov/go-fuzz)/[Atheris](https://github.com/google/atheris)
+/[Jazzer](https://github.com/CodeIntelligenceTesting/jazzer)).
```

Comment:
Add Java as supported language in libCASR docs below

Reply:
The same in libcasr module docs

Reply:
done